### PR TITLE
Lwdev 10289 LyCI API

### PR DESCRIPTION
**Pull request by AlexeyZelenin**

### CODE REVIEW

File: Core/Core.csproj [hunk 1]
```diff
@@ -20,5 +21,6 @@
     <PackageReference Include="Lykke.Service.ExchangeOperations.Client" Version="1.2.2" />
     <PackageReference Include="Lykke.Service.SwiftCredentials.Client" Version="1.0.0.8" />
     <PackageReference Include="Lykke.Service.ConfirmationCodes.Client" Version="1.1.3" />
+    <PackageReference Include="Refit" Version="4.6.58" />
```
Comment (polonzer) [inline Core/Core.csproj]: Why do we need it here?

File: LykkeApi2/LykkeApi2.csproj [hunk 2]
```diff
@@ -51,10 +52,11 @@
     <PackageReference Include="Lykke.Job.HistoryExportBuilder.Contract" Version="1.0.8" />
     <PackageReference Include="Lykke.SlackNotification.AzureQueue" Version="2.0.5" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.3.0" />
-    <PackageReference Include="Lykke.SettingsReader" Version="4.0.1" />
+    <PackageReference Include="Lykke.SettingsReader" Version="4.2.0" />
     <PackageReference Include="MessagePack" Version="1.7.3.4" />
     <PackageReference Include="Microsoft.AspNetCore" Version="2.1.2" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="2.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.ResponseCaching" Version="2.2.0" />
```
Comment (polonzer) [inline LykkeApi2/LykkeApi2.csproj]: Why do we need it here?

File: LykkeApi2/Controllers/IndicesController.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                return BadRequest(string.Format(Phrases.FieldShouldNotBeEmptyFormat, nameof(assetId)));
+
+            Index result;
+            try
+            {
+                result = await _indicesFacadeClient.Api.GetAsync(assetId);
+            }
+            catch (ClientApiException)
+            {
+                return NotFound();
```
[the remark this answers is not in the document]
Comment (AlexeyZelenin) [reply LykkeApi2/Controllers/IndicesController.cs]: no

File: LykkeApi2/Controllers/IndicesController.cs [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+using Lykke.Common.ApiLibrary.Exceptions;
+using Lykke.Service.IndicesFacade.Client;
+using Lykke.Service.IndicesFacade.Contract;
+using LykkeApi2.Strings;
+
+namespace LykkeApi2.Controllers
+{
+    [Route("api/indices")]
+    [ApiController]
+    public class IndicesController : Controller
```
[the remark this answers is not in the document]
Comment (AlexeyZelenin) [reply LykkeApi2/Controllers/IndicesController.cs]: added

File: LykkeApi2/Controllers/IndicesController.cs [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+                return BadRequest(string.Format(Phrases.FieldShouldNotBeEmptyFormat, nameof(timeInterval)));
+
+            IList<HistoryElement> result;
+            try
+            {
+                result = await _indicesFacadeClient.Api.GetHistoryAsync(assetId, timeInterval);
+            }
+            catch (ClientApiException)
+            {
+                return NotFound();
```
[the remark this answers is not in the document]
Comment (AlexeyZelenin) [reply LykkeApi2/Controllers/IndicesController.cs]: no

File: Core/Core.csproj (hunk 1 above)
Comment (AlexeyZelenin) [reply Core/Core.csproj]: removed

File: LykkeApi2/LykkeApi2.csproj (hunk 2 above)
Comment (AlexeyZelenin) [reply LykkeApi2/LykkeApi2.csproj]: removed

**Pull request merged**